app/views/movements/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div>
+  <div class="loginHeader groupHeader">
+    <%= link_to group_movements_path(@group), class: 'backArrow' do %>
+      <i class="icon-arrow-left"></i>
+    <% end %>
+    <h2 class="groupHeaderTitle titleT">Add a new transaction</h2>
+  </div>
+  <h4 class="noMovements"> All fields are mandatory </h4>
+  <%= form_with model: @movement, url: group_movements_path(@group), data: { turbo: false }, class: 'add-form' do |form| %>
+
+    <div class="field">
+      <%= form.text_field :name, placeholder: 'Enter the name' %>
+    </div>
+
+    <div class="field">
+      <%= form.number_field :amount, placeholder: 'Enter the amount' %>
+    </div>
+
+    <div class="field">
+      <%= form.hidden_field :group_id, value: @group.id %>
+    </div>
+
+    <div class="action">
+      <%= form.submit 'Save', class: "saveT"%>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [OPTIONAL] Based on the requirements, when a user wants to fill in information for a new transaction, the user should be able to fill out the name of the transaction, the amount and at least one of the categories that exist. I would advise you to implement this in your design of the transaction form. 👍 😎 

![image](https://github.com/federicaulzurrun/PennyLane/assets/77106962/96ccb030-e068-46c7-b0e1-d35eb77843bd)